### PR TITLE
Feature/apc dictionary

### PR DESCRIPTION
**Pull request by metelik**
Hello,

Herewith, We'd like to propose addition of APC's Schneider Electric's RADIUS dictionary - similar to one in the FreeRadius (ref.  https://github.com/FreeRADIUS/freeradius-server/blob/master/share/dictionary/radius/dictionary.apc ) .
We shall have been proposing amendments to the FreeRADIUS one as well.

With my best wishes
     Tomasz Motyl

### CODE REVIEW

File: priv/dictionaries/dictionary.apc [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#
+
+VENDOR APC 318
+
+BEGIN-VENDOR APC
+#
+# Attributes
+#
+ATTRIBUTE	APC-Service-Type	1	  integer
+ATTRIBUTE	APC-Outlets				2	  string
```
Comment (vkatsuba) [inline priv/dictionaries/dictionary.apc:15]: Please, align text in all lines

**Comment by vkatsuba** [comment]
Hi @metelik, thanks for contribution! Could you please squash commints into single commit? Thanks.

Regards,,
--V

**Comment by vkatsuba** [comment]
Also @metelik, if you can please rebase with master and try run actions in for your branch. Was added `workflow_dispatch` for this case. You can do it after rebasing like:
* Go to Action tab in fork
* Select `CI`
* Should be showed text `This workflow has a workflow_dispatch event trigger.` And click to button with text `Run workflow`

Thanks.

Regards,
--V

**Comment by vkatsuba** [comment]
@metelik all is ok, Slack error is expected for `forks`. Please, squash all commits into one single commit.

Regards, 
--V

**Comment by metelik** [comment]
> 
> 
> @metelik all is ok, Slack error is expected for `forks`. Please, squash all commits into one single commit.
> 
> Regards,
> --V

I declare a defeat on this one.

**Comment by vkatsuba** [comment]
Moved to https://github.com/travelping/eradius/pull/199/files

**Pull request closed without merging**